lambda-code/nagware/src/lib/redisConnector.ts

```diff
@@ -1,3 +1,8 @@
+import AWSXRay from "aws-xray-sdk-core";
+// Need to wrap the http module globally before importing it
+// to ensure that all outgoing HTTP requests are captured by AWS X-Ray.
+AWSXRay.captureHTTPsGlobal(require("http"));
+
```

Comment:
(Same problem I reported in the next comment on plugging `AWSXRay.captureHTTPsGlobal`)

I am guessing you wanted to capture any Redis traffic that would be based on the `http` library. I tested it with my fix in place and I could not see anything in XRay. 

I then tried to plug XRay in `emailNotification.ts` as a test (I figured you did not add it in this PR because we could plug XRay directly in the `connectors` package). I was able to see the call to the API in XRay traces.

Note: since this file is imported before we send requests to GCNotify it allows calls to their API to be caught by XRay. Maybe we could do such initialization in the `main` files.